index.html

```diff
@@ -10,23 +10,63 @@
 </head>
```

Comment:
Kindly work on these requested changes:
- Fix your linter issues in your pull request.
- Add a live version on your project deployed in your README file.

Reply:
Thank you and noted the feedback.
Working on it.